test/test_autotokenizer.py

```diff
@@ -69,6 +69,45 @@ def test_clip_tokenizer(self):
         actual_ids = ort_tok([text])[0]
         np.testing.assert_array_equal(ids, actual_ids)
 
+    def test_xmlroberta_tokenizer(self):
```

Comment:
nit: typo - should be xlmroberta not xml

Reply:
let's fix it the next PR.